src/devtools/client/debugger/src/reducers/pause.ts

```diff
@@ -283,7 +283,11 @@ async function getResumePoint(replayClient: ReplayClientInterface, state: UIStat
   if (!frames || !frame || frame === frames[0]) {
     return;
   }
-  const frameSteps = await getFrameStepsAsync(selectedFrameId.pauseId, selectedFrameId.frameId);
+  const frameSteps = await getFrameStepsAsync(
```

Comment:
I'm a little confused about why this function now calls both `getFramesAsync` and `getFrameStepsAsync`. Seems like we can remove the `getFramesAsync` call now? We only really need the steps?

Reply:
We still check if the currently selected frame is the top frame and bail out in that case.

Reply:
D'oh. I misread that. Ok.